recipes/rb-connection_pool/LICENSE.txt

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2011 Mike Perham
```

Comment:
@svengiegerich, you should be able to get rid of the license file by following the same steps we discussed here: https://github.com/conda-forge/staged-recipes/pull/22205